src/test/java/org/opensrp/web/utils/SearchHelperTest.java

```diff
@@ -91,7 +79,7 @@ public void testGetContactPhoneNumberParamReturnsCorrectValueForParam() {
 	@Test
 	public void testGetChildIndentifierFromMotherClient() {
 		
-		String childSystemId = SearchHelper.getChildIndentifierFromMotherClient(testClient, "M_ZEIR_ID", "mother");
+		String childSystemId = SearchHelper.getChildIdentifierFromMotherClient(testClient, "M_ZEIR_ID", "mother");
```

Comment:
Maybe we don't need this method any more, we can remove both the test and it's implementation

Reply:
yeah, removed it